osuosl/climesync
# Cohenaa/validate inputs

fixes issue #113 
## Changes in this PR.


- [X] Cache project slugs, activity slugs, usernames, and user info on login
- [X] Use that data to validate fields where one or more slugs/usernames are entered and ask the user to try again
- [X] If there are no valid projects/activities/etc. to be entered, exit from the command
## Testing this PR.


1. Run `nosetests`
2. Run `climesync/climesync.py`, connect to the staging server, and log in as `admin`/`timesync-staging`
3. Run `gt` and attempt to enter invalid users, projects, and activities. See that you are given an error and asked to enter valid responses

@osuosl/devs
